helpers/src/data.js

```diff
@@ -0,0 +1,1822 @@
+module.exports = {
```

Comment:
How is this file generated?

Reply:
+1,1822 OMG

Reply:
dont' commit it: it is built by "publish" of the package.
(also, I'd put it in a "build" folder, not in "src")
why do you change the json file to JS ? you can "require" json files just as well, and thus use truffle compiler output as-is (just move folder)

Reply:
I took the oz approach of compiling it and putting it in data.js file, you're free to change it as well.